workers/python/kitchen_sink.py

```diff
@@ -104,16 +111,55 @@ async def handle_action(self, action: Action) -> Optional[Payload]:
         elif action.HasField("continue_as_new"):
             workflow.continue_as_new(action.continue_as_new.arguments)
         elif action.HasField("timer"):
-            await asyncio.sleep(action.timer.milliseconds / 1000)
+            await handle_awaitable_choice(
+                asyncio.sleep(action.timer.milliseconds / 1000),
+                action.timer.awaitable_choice,
+            )
         elif action.HasField("exec_activity"):
-            await launch_activity(action.exec_activity)
+            await handle_awaitable_choice(
+                launch_activity(action.exec_activity),
+                action.exec_activity.awaitable_choice,
+            )
         elif action.HasField("exec_child_workflow"):
             child_action = action.exec_child_workflow
             child = child_action.workflow_type or "kitchenSink"
             args = [RawValue(i) for i in child_action.input]
-            await workflow.execute_child_workflow(
-                child, id=child_action.workflow_id, args=args
+            await handle_awaitable_choice(
+                workflow.start_child_workflow(
+                    child, id=child_action.workflow_id, args=args
+                ),
+                child_action.awaitable_choice,
             )
+        elif action.HasField("set_patch_marker"):
+            if action.set_patch_marker.deprecated:
```

Comment:
How is this supposed to work with Go and Java?

Reply:
I really don't think it's an issue - keep in mind these tests are not about verifying behavioral specifics. All that matters here is that we generate the patch/versioning command (marker).